app/services/build_defect.rb

```diff
@@ -18,7 +20,14 @@ def call
     defect.set_target_completion_date
 
     set_created_at if created_at.present?
+    set_evidence_user
 
     defect
   end
+
+  private
+
+  def set_evidence_user
+    defect.evidences.first.user = user if defect.evidences.first.present?
```

Comment:
Could we do `defect.evidences.present?` here?

Reply:
I'd used first to mimic the setting we're doing, but since this is a builder we can safely assume there will be no `evidences`, so happy to make this change.